cmd/joy/release.go

```diff
@@ -384,17 +385,25 @@ func NewGitCommands() *cobra.Command {
 					return fmt.Errorf("getting target tag from release: %w", err)
 				}
 
-				gitargs := append([]string{"git", command}, args[1:]...)
-				gitargs = append(gitargs, sourceTag+".."+targetTag)
+				gitArgs := append([]string{"git", command}, args[1:]...)
+				gitArgs = append(gitArgs, sourceTag+".."+targetTag)
 
-				fmt.Println(color.InCyan("running:"), color.InBold(strings.Join(gitargs, " ")))
+				if project.Spec.RepositorySubPaths != nil {
+					if slices.Contains(gitArgs, "--") {
+						gitArgs = append(gitArgs, "--")
+					}
+					gitArgs = append(gitArgs, project.Spec.RepositorySubPaths...)
+				}
+
+				fmt.Println(color.InCyan("running:"), color.InBold(strings.Join(gitArgs, " ")), color.InCyan("in"), color.InBold(repoDir))
 				fmt.Println()
 
-				gitCommand := exec.Command("git", gitargs[1:]...)
+				gitCommand := exec.Command("git", gitArgs[1:]...)
 				gitCommand.Dir = repoDir
 				gitCommand.Stdout = os.Stdout
 				gitCommand.Stderr = os.Stderr
 				gitCommand.Stdin = os.Stdin
+				gitCommand.Env = os.Environ()
```

Comment:
Otherwise, I can't hijack stuff like the `GIT_PAGER.`

Reply:
Are you sure? If its not present it defaults to os.Environ:
```
Env specifies the environment of the process. Each entry is of the form "key=value". If Env is nil, the new process uses the current process's environment
```